src/busterbunny.js

```diff
@@ -212,9 +212,16 @@ module.exports = (function() {
                 config.queues.forEach(function(queue) {
                     channel.assertQueue(queue.name);
                     channel.consume(queue.name, function(message) {
+                        var event;
                         try {
-                            var event = self.encoder.decodeEvent(message);
+                            event = self.encoder.decodeEvent(message);
+                        } catch(err) {
+                            channel.nack(message, false, false);
+                            _stats.messagesRejectedWithRetry++;
+                            self.emit(self.EVENTS.EVENT_NACKED, null, message, false, new Date().getTime());
+                        }
 
+                        if(event) {
```

Comment:
because you've moved the closure of the try block to only include the decoding of the event, a secondary try should be added to this reject/acknowledge section.